jfix-armeria-facade/src/main/kotlin/ru/fix/armeria/facade/webclient/impl/BaseHttpClientBuilderImpl.kt

```diff
@@ -58,53 +56,41 @@ internal abstract class BaseHttpClientBuilderImpl<out HttpClientBuilderT : BaseH
     )
 
     override fun setEndpoint(uri: URI) = copyOfThisBuilder(
-        baseBuilderState.copy(endpointGroupCreator = {
-            Either.Right(uri)
-        })
+        baseBuilderState.copy(endpoint = uri)
```

Comment:
please fix BaseHttpClientBuilder javadocs after making endpoint declaration optional (there is a block `Mandatory builder methods. Client name and endpoint/endpoint_group must be formed.`)